lib/assessment.rb

```diff
@@ -13,28 +13,32 @@ def longest(enum)
   # enumerable supplied (i.e. apply the `+` operator)
   # Assume each entry implements a `+` method
   def sum(enum)
-    # Your code goes here
+    reduce (memo, f) { return memo + f }
   end
 
   # Question 3
   # Write ruby code that returns true if some of the entries
   # in the numbers enumerable are odd
+
+  # Yeah... confused
   def some_odd(numbers)
-    # Your code goes here
+    each_with_index { |number, index| number.odd? }
```

Comment:
`odd?` is the right tool, but you used the wrong enumerable method. Which one would we use if we wanted to determine if at least some elements in our collection passed a particular test?